BrowserKit/Sources/ToolbarKit/AddressToolbar/AddressToolbarAddTabView.swift

```diff
@@ -26,19 +23,25 @@ final class AddressBarAddTabView: UIView, ThemeApplicable {
     }
 
     private func setupLayout() {
-        layer.cornerRadius = UX.cornerRadius
         addSubview(plusIconView)
         NSLayoutConstraint.activate([
             plusIconView.centerXAnchor.constraint(equalTo: centerXAnchor),
             plusIconView.centerYAnchor.constraint(equalTo: centerYAnchor)
         ])
     }
 
+    public func config(_ configuration: AddressToolbarUXConfiguration) {
```

Comment:
Can we rename this to `configure` to be match what we have in other places in the project?